recipe/meta.yaml

```diff
@@ -61,13 +79,19 @@ outputs:
       requires:
         - pip
         - pytest
+        - mypy
+        - attrs
+        - pyyaml
+        - tomli    # [py<311]
+        - tomli-w
+        - msgpack-python
       source_files:
         - tests
-        - msgspec/_core.c
-        - setup.cfg
+        - src/msgspec/_core.c
+        - pyproject.toml
       commands:
         - pip check
-        - pytest tests
+        - pytest -ra -v {{ ignore_tests_files }} tests  # [not (win and py==310) and not (win and py==311)]
```

Comment:
You can drop these tests entirely, because they duplicate the `msgspec` subpackage's tests above